### ISSUE TITLE
could not open shared memory segment  error in v46

### ISSUE THREAD

**Issue by jpalermo** [cloudfoundry/postgres-release#73]
`psql: error: connection to server at "127.0.0.1", port 5432 failed: FATAL:  could not open shared memory segment "/PostgreSQL.231763232": No such file or directory`

The issue seems to be documented [here](https://stackoverflow.com/questions/56954665/postgresql-11-shared-memory-error-could-not-open-shared-memory-segment-postgr) and can be worked around by adding `RemoveIPC=no` to `/etc/systemd/logind.conf` and then `systemctl restart systemd-logind.service`

Not sure if we want to add that to the pre-start script, or figure out a better way to solve this problem.

**Comment by jpalermo** [comment]
We've got a [PR](https://github.com/cloudfoundry/bosh-linux-stemcell-builder/pull/297) in progress for jammy stemcells to resolve the issue there.

For now, you can work around this by using the `pre-start-script` job of os-conf-release and using this as the script property:
```
echo 'RemoveIPC=no' >> /etc/systemd/logind.conf
systemctl restart systemd-logind.service
```

**Comment by jochenehret** [comment]
Automatic update of the postgres-release is currently failing: https://concourse.wg-ard.ci.cloudfoundry.org/teams/main/pipelines/update-releases/jobs/update-postgres/builds/6
We'll wait for the proposed fix and then try again.

**Comment by jpalermo** [comment]
Looks like the fix in Jammy 1.222 has resolved the problem.
